### PR TITLE
fix(codelab): Update k8s codelab images & links

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

File: guides/tutorials/codelabs/kubernetes-source-to-prod/index.md [hunk 1]
```diff
@@ -50,17 +50,21 @@ Follow one of the guides [here](http://kubernetes.io/docs/getting-started-guides
...
 
-![Applications data and spin were created by Spinnaker, the rest were created by Kubernetes.](applications.png)
+Applications data and spin were created by Spinnaker, the rest were created by Kubernetes.
+
+![](applications.png)
 
 Under the _Actions_ dropdown select _Create Application_ and fill out the following dialog:
 
-![If you've followed the Source to Prod tutorial for the VM based providers, you'll remember that you needed to select "Consider only cloud provider health when executing tasks". Since Kubernetes is the sole health provider by definition, selecting this here is redundant, and unnecessary.](appfill.png)
+If you've followed the Source to Prod tutorial for the VM based providers, you'll remember that you needed to select "Consider only cloud provider health when executing tasks". Since Kubernetes is the sole health provider by definition, selecting this here is redundant, and unnecessary.
```
Comment (lwander) [inline guides/tutorials/codelabs/kubernetes-source-to-prod/index.md:68]: @duftler these are the kinds of text/image combos that aren't rendered after the migration from readme

**Comment by lwander** [comment]
ping

File: guides/tutorials/codelabs/kubernetes-source-to-prod/index.md [hunk 2]
```diff
@@ -27,10 +27,10 @@ The workflow generally looks like this:
 
 We need a few things to get this working. 
 
-  1. [A Github repo containing the code we want to deploy](http://www.spinnaker.io/v1.0/docs/kubernetes-source-to-prod#section-configuring-github).
-  2. [A Dockerhub repo configured to build on changes to the above repo](http://www.spinnaker.io/v1.0/docs/kubernetes-source-to-prod#section-configuring-dockerhub).
-  3. [A running Kubernetes cluster](http://www.spinnaker.io/v1.0/docs/kubernetes-source-to-prod#section-configuring-kubernetes).
-  4. [A running Spinnaker deployment configured with the contents of steps 2 and 3](http://www.spinnaker.io/v1.0/docs/kubernetes-source-to-prod#section-configuring-kubernetes).
+  1. [A Github repo containing the code we want to deploy](#configuring-github).
+  2. [A Dockerhub repo configured to build on changes to the above repo](#ction-configuring-dockerhub).
```
Comment (ttomsu) [inline guides/tutorials/codelabs/kubernetes-source-to-prod/index.md]: Looks like a typo in the fragment

File: guides/tutorials/codelabs/kubernetes-source-to-prod/index.md [hunk 3]
```diff
@@ -50,17 +50,21 @@ Follow one of the guides [here](http://kubernetes.io/docs/getting-started-guides
 
 ## Configuring Spinnaker
 
-We will be deploying Spinnaker to the same Kubernetes cluster it will be managing. To do so, follow the steps in [this guide](https://github.com/spinnaker/spinnaker/tree/master/experimental/kubernetes/simple), being sure to use [this section](https://github.com/spinnaker/spinnaker/tree/master/experimental/kubernetes/simple/#anything-else-except-for-ecr) to configure your registry.
+We will be deploying Spinnaker to the same Kubernetes cluster it will be managing. To do so, follow the installation [guide](/setup/install), being sure to enable & create a Kubernetes account, as well as a Docker Registry account with credentials and repositories pointing at those created above
```
Comment (ttomsu) [inline guides/tutorials/codelabs/kubernetes-source-to-prod/index.md]: Need period at end.

File: guides/tutorials/codelabs/kubernetes-source-to-prod/index.md [hunk 4]
```diff
@@ -50,17 +50,21 @@ Follow one of the guides [here](http://kubernetes.io/docs/getting-started-guides
...
 
 # 1. Create a Spinnaker Application
 
 Spinnaker applications are groups of resources managed by the underlying cloud provider, and are delineated by the naming convention `<app name>-`. Since Spinnaker and a few other Kubernetes-essential pods are already running in your cluster, your _Applications_ tab will look something like this:
 
-![Applications data and spin were created by Spinnaker, the rest were created by Kubernetes.](applications.png)
+Applications data and spin were created by Spinnaker, the rest were created by Kubernetes.
```
Comment (ttomsu) [inline guides/tutorials/codelabs/kubernetes-source-to-prod/index.md]: Is this supposed to be a caption? If so, take a look at  https://github.com/spinnaker/spinnaker.github.io/blob/master/guides/tutorials/codelabs/halyard-getting-started/index.md for an example of using `{% include figure %}` which will do the captioning for you.

**Review by ttomsu — APPROVED** [review]

**Pull request merged**